optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -521,6 +522,11 @@ def initialize(self, config):
                                  % (entry))
             elif len(entry) == 2:   # key, entry
                 entry.insert(0, '')  # add path='' (root)
+            # We try to maintain the data type of the provided value
+            try:
+                entry[2] = ast.literal_eval(entry[2])
+            except ValueError:
+                entry[2] = entry[2]
```

Comment:
Unless I'm missing something, this makes no sense to me. A simple `pass` with a warning/debug message wouldn't be better here?

Reply:
This gives a way to provide input to be injected as `bool`, `int` or `string`.. The existing code treats any injected value as a string.

Reply:
I'm talking about line 529 exactly.

Reply:
@beraldoleal Agreed.

Reply:
@beraldoleal Do we really need debug log here ? I am thinking of just `pass`ing

Reply:
While I'm ok with a single `pass`, a debug log message would be helpful to understand what happened.

Reply:
@beraldoleal I am finding it difficult to draft a debug message, since that debug message is going to explain the behaviour which was default thus far. And none of the optional plugins have been writing to the job.log. So, unless you are very specific about debug message and have some suggestions, I am `pass`ing.

Reply:
Done. Please check.